demos/include/iot_demo_runner.h

```diff
@@ -130,6 +130,8 @@
     #define DEMO_entryFUNCTION             vRunCLIUartDemo
 #elif defined( CONFIG_DEVICE_DEFENDER_DEMO_ENABLED )
     #define DEMO_entryFUNCTION             RunDeviceDefenderDemo
+#elif defined( CONFIG_OTA_MQTT_UPDATE_DEMO_ENABLED )
+    #define DEMO_entryFUNCTION             RunOtaCoreMqttDemo
```

Comment:
Minor thing but what about naming this as RunOtaCoreMqttAgent demo ?

Reply:
NIT: We could but we are still using coreMQTT under the hood, also the http demo entry function would be `RunOtaCoreHttpDemo` hence could keep it the same for consistency?